### PR TITLE
Added simple docx loader - docx2txt

### PR DESCRIPTION
**Pull request by rishni-firstthing-ai**
Added a simple docx loader for those that don't have/use Unstructured

### CODE REVIEW

File: langchain/document_loaders/word_document.py [hunk 1]
```diff
@@ -1,10 +1,87 @@
...
+
+import requests
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
 from langchain.document_loaders.unstructured import UnstructuredFileLoader
 
 
+class BaseDOCXLoader(BaseLoader, ABC):
+    """Base loader class for PDF files.
```
Comment (hwchase17) [inline langchain/document_loaders/word_document.py:17]: docstring wrong - is not PDF?

**Review by hwchase17 — COMMENTED** [review]
would be great to add this somewhere to example docs!

File: langchain/document_loaders/word_document.py (hunk 1 above)
Comment (rishni-firstthing-ai) [reply langchain/document_loaders/word_document.py:17]: Oh. Of course.

**Comment by rishni-firstthing-ai** [comment]
Ok. I'll throw in an example, fix the docstring comment, and push again.

N00b question: Maybe someone can tell me why the lint / build step keeps failing? What do I need to fix?

**Comment by hwchase17** [comment]
> Ok. I'll throw in an example, fix the docstring comment, and push again.
> 
> N00b question: Maybe someone can tell me why the lint / build step keeps failing? What do I need to fix?

if you run `make format` and then `make lint` locally you should see more informative error mesages. im also happy to fix those!

**Comment by rishni-firstthing-ai** [comment]
The failure on the lint/build doesn't look like something I can fix?

$ make lint
poetry run mypy .
venv\Scripts\activate_this.py:27: error: "str" has no attribute "decode"; maybe "encode"?  [attr-defined]
venv\Scripts\activate_this.py:30: error: Module has no attribute "real_prefix"  [attr-defined]
Found 2 errors in 1 file (checked 717 source files)
make: *** [lint] Error 1

**Pull request merged**